### PR TITLE
Added all the changes for cancer classifier

### LINKED ISSUE #10: Create a Random Forest Classifier to predict Breast Cancer using Numerical Data

### PR DESCRIPTION
**Pull request by ckmganesh**
Fixes #10 
Creating a new PR as something went wrong in the other one
Added all the changes suggested and also changed the branch for PR

### CODE REVIEW

File: Model/randomforest_cancerclassifier.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+print("Total test observations :",Y_pred.shape[0] )
+print()
+#Confusion Matrix
+from sklearn.metrics import confusion_matrix
+cm = confusion_matrix(Y_test, Y_pred)
+print(cm)
+print()
+##Accuracy
+print("Accuracy :",(cm[0][0]+cm[1][1])/Y_pred.shape[0])
+print()
```
Comment (srimani-programmer) [inline Model/randomforest_cancerclassifier.py:53]: @ckmganesh please add the extra line at the end of the file.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]

File: Model/randomforest_cancerclassifier.py (hunk 1 above)
Comment (ckmganesh) [reply Model/randomforest_cancerclassifier.py:53]: I don't understand.should I just add a line?

Comment (srimani-programmer) [inline Model/randomforest_cancerclassifier.py:53]: @ckmganesh please remove the extra print statement and just add an empty line.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]

**Comment by honey2158** [comment]
I have also written a code using 3 different models and got a good accuracy where should i upload my code. As I am new to open source and don't know much about github can anyone help me with it.
